backends/intel_gpu/load.sh

```diff
@@ -0,0 +1,20 @@
+
+
+
+PaddleDev=$(dirname $(dirname `pwd`))
+
+echo $d
+export PYTHONPATH=$PYTHONPATH:${PaddleDev}/python/tests/
+
+
+comp="dnnl tbb compiler dpl"
+
+for item in $comp;
+do
+
+	P="${HOME}/intel/oneapi/$item/latest/env/vars.sh"
+        echo "$P"
+        source $P
+done
+
```

Comment:
this is an debug scripts? Please remove

Reply:
we need this file, the script is loading variables for correct usage before first run